.github/scripts/delete_old_branches.py

```diff
@@ -64,7 +64,7 @@ def main(branch_age_limit_days: int, dry_run: bool) -> None:
         print(log_prefix + 'Did not delete any branches.')
 
 
-if __name__ == "__main__":
+if __name__ == "__main__":  # pragma: no cover
```

Comment:
what does the 'pragma: no cover mean'?

Reply:
[This wikipedia article](https://en.wikipedia.org/wiki/Directive_(programming)) explains what a pragma is in general. This particular pragma is intended for use with pytest code coverage as explained [in this documentation](https://coverage.readthedocs.io/en/6.4.4/excluding.html).

Reply:
Thanks for explaining